configure.ac

```diff
@@ -3821,6 +3821,40 @@ if test "x$with_libnetsnmp" = "xyes"; then
   LDFLAGS="$SAVE_LDFLAGS"
 fi
 
+if test "x$with_libnetsnmp" = "xyes"; then
+  SAVE_LDFLAGS="$LDFLAGS"
+  LDFLAGS="$LDFLAGS $with_libnetsnmp_ldflags"
+
+  AC_CHECK_LIB([netsnmp], [netsnmp_get_version],
+    [with_libnetsmp="yes"],
+    [with_libnetsnmp="no (couldn't get libnetsnmp version)"]
+  )
+
+  LDFLAGS="$SAVE_LDFLAGS"
+fi
+
+if test "x$with_libnetsnmp" = "xyes"; then
+  SAVE_CPPFLAGS="$CPPFLAGS"
+  CPPFLAGS="$CPPFLAGS $with_libnetsnmp_cppflags"
+
+  AC_LANG_CONFTEST([
+     AC_LANG_PROGRAM(
+       [[#include <stdio.h>
+         #include <net-snmp/net-snmp-config.h>
+         #include <net-snmp/net-snmp-includes.h>]],
+       [[fputs(netsnmp_get_version(), stdout);]]
+     )
+  ])
+
+  $CC -o conftest conftest.c $CPPFLAGS -lnetsnmp 2> /dev/null
+  NETSNMP_VER=`./conftest`
+  rm -f conftest
+  AX_COMPARE_VERSION([$NETSNMP_VER],[ge],[5.4.5],
```

Comment:
Why do you need this version or higher? Can't you test for functionality instead?

Reply:
Version below 5.4.5 has different API, there are some minor changes

Reply:
now testing for functionality